doc/user_manual/postprocessor.tex

```diff
@@ -2190,3 +2191,38 @@ \subsubsection{ValueDuration}
  ...
 </Simulation>
 \end{lstlisting}
+
+
+%%%%%%%%%%%%%% SampleSelector %%%%%%%%%%%%%%%%%%%
+\subsubsection{SampleSelector}
+\label{SampleSelectorPP}
+The \xmlNode{SampleSelector} postprocessor is a tool to select the row in which the minimum or the maximum of a target is found.
+
+The \xmlNode{SampleSelector} postprocessor can  act on \xmlNode{DataObjects}, and generates a
+\xmlNode{DataObject} in return.
```

Comment:
Does this handle PointSet, HistorySet and DataSet? (I think just the first 2... you should indicate so)

Reply:
Looking at the source code, I correct my self. It is applicable to any dataobject.